llvm/runtimes/CMakeLists.txt

```diff
@@ -208,6 +208,36 @@ foreach(entry ${runtimes})
   list(APPEND RUNTIME_NAMES ${name})
 endforeach()
 
+# Set LLVM_RUNTIME_<project>_BUILD variables if a sub-project is enabled to be built as
+# a runtime.  As with LLVM_TOOL_<project>_BUILD, the LLVM_RUNTIME_<project>_BUILD variables
+# should be not directly used from the CMake configuration command line.
+foreach(proj ${LLVM_SUPPORTED_RUNTIMES})
```

Comment:
Don't we need to check for each `LLVM_RUNTIME_TARGETS` if `RUNTIMES_<triple>_LLVM_ENABLE_RUNTIMES` is set?

Reply:
Good question.  I'm not qualified to answer it, though.